src/backend/base/langflow/components/retrievers/SelfQueryRetriever.py

```diff
@@ -62,7 +62,7 @@ def build(
             input_text = query
         else:
             raise ValueError(f"Query type {type(query)} not supported.")
-        documents = self_query_retriever.invoke(input=input_text)
+        documents = self_query_retriever.invoke(input=input_text)  # type: ignore
```

Comment:
same issue as above here - it's expected `input: str`. `input_text` can be a `Message (or an iterator)`. Not sure how you want to fix this one up easily